Adaptors/BDMMintegralAdapter/Adapters/BDMMintegralFullscreenAdapter.m

```diff
@@ -0,0 +1,143 @@
+//
+//  BDMMintegralFullscreenAdapter.m
+//  BDMMintegralAdapter
+//
+//  Created by Yaroslav Skachkov on 8/16/19.
+//  Copyright © 2019 Stas Kochkin. All rights reserved.
+//
+
+#import "BDMMintegralFullscreenAdapter.h"
+#import "BDMMintegralValueTransformer.h"
+#import <MTGSDKReward/MTGBidRewardAdManager.h>
+#import <MTGSDKInterstitialVideo/MTGBidInterstitialVideoAdManager.h>
+#import <MTGSDK/MTGSDK.h>
+#import <MTGSDKBidding/MTGBiddingRequest.h>
+
+@interface BDMMintegralFullscreenAdapter() <MTGRewardAdLoadDelegate, MTGRewardAdShowDelegate, MTGBidInterstitialVideoDelegate>
+
+@property (nonatomic, strong) MTGBidInterstitialVideoAdManager *ivBidAdManager;
+@property (nonatomic, strong) NSString *unitId;
```

Comment:
String property should be copy instead of strong

Reply:
+